python/lsst/daf/butler/formatters/pickleFormatter.py

```diff
@@ -55,7 +55,7 @@ def _readFile(self, path, pytype=None):
         """
         try:
             with open(path, "rb") as fd:
-                data = pickle.load(fd)
+                data = pickle.load(fd, encoding='latin1')
```

Comment:
I thought this was only needed to read old pickle files written in python 2?

Reply:
The obs_subaru brighter fatter kernel is a python2 pickle.

Reply:
If you can't do that I'd rather we had a separate formatter for reading legacy files.

Reply:
Is it possible to pass an optional encoding argument?

Reply:
The formatter is only designed to read in pickles that were written by the formatter. It's not really meant for reading in items created in other places. Did this pickle file get added to the repository via the back door?

The documentation suggests that encoding is only used if a python2 pickle file is found. I'd prefer a separate formatter explicitly for these legacy files.

Reply:
I've removed the formatter change, in favor of having an external tool to read and rewrite the obs_subaru bfKernel for this test.  As this file needs to be copied and manually inserted into the registry for Gen3 tests anyway, this isn't too much hassle, and avoids this issue for now.